packages/Confirmation/test/confirmation.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { render, configure } from "@testing-library/react";
+import L10n from "@paprika/l10n";
+import Confirmation from "../src";
+
+configure({ testIdAttribute: "data-qa-anchor" });
+
+function renderComponent() {
+  const renderedComponent = render(
+    <L10n>
+      <Confirmation defaultIsOpen confirmLabel="Confirm Delete" onConfirm={() => {}} heading="Delete Button?" />
+    </L10n>
+  );
+
+  return {
+    ...renderedComponent,
+  };
+}
+
+// function renderComponentWithTrigger() {
```

Comment:
Can we remove this? If you're saving it for later, a comment that explains why would be helpful.

Reply:
... just in case someone else takes it over and writes the additional tests...